### DISCUSSION TITLE
How to connect to anthropic claude models?

### DISCUSSION THREAD

**Discussion by Jasha10** [huggingface/llm.nvim#101] [Q&A]
I'd like to use claude 3.5 sonnet for code completion in Neovim. Is there a straightforward way to configure this using llm.nvim?

**Comment by McPatate** [comment]
> Is there a straightforward way to configure this using llm.nvim?


From skimming the docs, it appears they have their own API, so no straightforward way to configure llm.nvim to use claude.

We would have to implement the backend in https://github.com/huggingface/llm-ls.
